### PR TITLE
Replicate specification of Object extension into Storage Root extensi…

### PR DESCRIPTION
**Pull request by awoods**
…ons section

Resolves: https://github.com/OCFL/spec/issues/557

### CODE REVIEW

File: draft/spec/index.html [hunk 1]
```diff
@@ -1105,10 +1105,32 @@ <h2>Storage Root Extensions</h2>
...
+            purposes of extending the functionality of an OCFL Storage Root. The guidelines and limitations for the
+            storage root <code>extensions</code> directory are defined in alignment with those of the
             <a href="#object-extensions">object extensions</a>.
         </p>
+        <p>
+            The <code>extensions</code> directory <span id="E112">MUST NOT</span> contain any files, and no
+            sub-directories other than extension sub-directories. Extension sub-directories
+            <span id="W016">SHOULD</span> be named according to a <a>registered extension name</a>. The specific
+            structure and function of the extension, as well as a declaration of the registered extension name
+            <span id="E113">MUST</span> be defined in one of the following locations:
```
Comment (pwinckles) [inline draft/spec/index.html:1117]: Personally, I would not include `E113` it is impossible to enforce.

Comment (zimeon) [reply draft/spec/index.html:1117]: I agree that we should drop `E113` -- the "structure and function part" is not enforceable, and the notion of it being a registered name overlaps with the warning `W016`. (This also applies in the object extensions section for `E068` and `W013`.)

Comment (zimeon) [reply draft/spec/index.html:1117]: I agree that we should drop `E113` -- the "structure and function part" is not enforceable, and the notion of it being a registered name overlaps with the warning `W016`. (This also applies in the object extensions section for `E068` and `W013`.)

Comment (zimeon) [reply draft/spec/index.html:1117]: I note this has tons of overlap with https://github.com/OCFL/spec/issues/565, will propose approach there. I think we could move forward with this PR (addressing the replication issue alone) by simply remove the text from the `E113` sentence onward.

**Pull request closed without merging**